workflow/rules/mapping.smk

```diff
@@ -28,6 +30,19 @@ rule merge_untrimmed_fastqs:
         "cat {input} > {output} 2> {log}"
 
 
+rule sort_untrimmed_fastqs:
+    input:
+        "results/untrimmed/{sample}_{read}.fastq.gz",
+    output:
+        temp("results/untrimmed/{sample}_{read}.sorted.fastq.gz"),
+    conda:
+        "../envs/fgbio.yaml"
+    log:
+        "logs/fgbio/sort_fastq/{sample}_{read}.log",
+    shell:
+        "fgbio SortFastq -i {input} -o {output} 2> {log}"
+
+
```

Comment:
I would prefer to remove this rule and perform sorting directly when merging fastqs in `merge_untrimmed_fastqs` to reduce IO.
Unfortunately piping does not seem to work. I tried `cat {input} | fgbio SortFastq -i /dev/stdin -o {output}` but fgbio keeps failing. Any suggestion how this could be combined?

Reply:
I've tried this but fgbio also fails with named pipes and even the `--async-io` flag